packages/mdc-toolbar/_mixins.scss

```diff
@@ -60,8 +60,9 @@
   position: relative;
   align-items: start;
   justify-content: center;
-  width: $mdc-toolbar-icon-size;
-  height: $mdc-toolbar-icon-size;
+  box-sizing: border-box;
+  width: $mdc-toolbar-icon-size * 2;
```

Comment:
Let's change this (and below) to `$mdc-toolbar-icon-size + $mdc-toolbar-icon-padding * 2` to be more clear?